.github/workflows/github-actions.yml

```diff
@@ -22,6 +22,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Set Up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+      - name: Install dependencies
+        run: npm install && 
```

Comment:
It might be simpler to just use `npm run build` instead of the next few lines here. The build script goes through and installs all the lambda layer libraries as well: https://github.com/eosdis-nasa/earthdata-pub-api/blob/develop/scripts/build.sh#L22-L26

Reply:
Made this change its way cleaner.